testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -73,8 +73,8 @@ class DockerComposeNamespace(DockerNamespace):
         COMPOSE_FILES_PATH + "/extra/recaptcha-testing/tenantadm-test-recaptcha-conf.yml",
     ]
 
-    NUM_SERVICES_OPENSOURCE = 14
-    NUM_SERVICES_ENTERPRISE = 18
+    NUM_SERVICES_OPENSOURCE = 11
+    NUM_SERVICES_ENTERPRISE = 13
```

Comment:
Maybe you know already, but I didn't see it, so just FYI: This needs to be changed in [mender-docs](https://github.com/mendersoftware/mender-docs/blame/master/07.Administration/02.Production-installation/docs.md#L654) as well. There are several of those, this is a good expression to find them:

```bash
git grep -e "grep -c -i 'up'"
```

Reply:
Ready to launch it, indeed.